src/components/BarChart/WalletChart.tsx

```diff
@@ -148,8 +150,11 @@ const WalletChart = ({ height }: Props) => {
                 paddingRight="m"
                 fontSize={16}
                 opacity={activityChartRange === 'daily' ? 1 : 0.3}
+                accessibilityLabel={t(
+                  'wallet.chartRanges.days.accessibilityLabel',
+                )}
```

Comment:
Can you add accessibilityRole="button" too?

Reply:
Will do. 👍